pipenv/patched/pip/_internal/vcs/subversion.py

```diff
@@ -288,17 +288,22 @@ def fetch_new(
             display_path(dest),
         )
         if verbosity <= 0:
-            flag = "--quiet"
+            cmd_args = make_command(
```

Comment:
This definitely looks like something that would not encounter  much resistance in upstream.